lib/src/widgets/message_list/message_list.dart

```diff
@@ -156,6 +161,7 @@ class MessageListState extends State<MessageList> {
                     .scrollToBottomBuilder!(scrollController)
```

Comment:
`scrollToBottomBuilder` could return whether it's `readOnly` or not so you can use it in your own custom builder... but the way we implemented it in our own custom build was just by reusing the variable that sets `readOnly` on `DashChat()`.